pkg/pillar/types/global.go

```diff
@@ -122,6 +122,9 @@ const (
 	MetricInterval GlobalSettingKey = "timer.metric.interval"
 	// DiskScanMetricInterval global setting key
 	DiskScanMetricInterval GlobalSettingKey = "timer.metric.diskscan.interval"
+	// CertExpireInfo and CertExpireWarn global thresholds (in days)
+	CertExpireInfo GlobalSettingKey = "timer.metric.certinfolog"
```

Comment:
We don't have a strict scheme for naming these, but the "metric" part doesn't make sense.  Something like timer.certcheck.infolog makes more sense as a naming approach.

Reply:
Fully agree! Done -- will merge RSN